### DISCUSSION TITLE
Anyone found a cheap small linux box to only run 86box?

### DISCUSSION THREAD

**Discussion by Thraka** [86Box/86Box#3228] [General]
I love messing around with 86box, and running ao486 on my Mister has got me thinking about having a small dedicated machine hooked up to the same tv as my Mister, and running 86box on it. It would be a great machine to build out different configs for and play games, explore other OS and classic dev tools, etc.

Has anyone already done this?

**Comment by parkerlreed** [comment]
Don't know about cheap but there's a ton of those tiny Intel NUCs

Would probably make for a nice little 86box machine.

https://www.youtube.com/watch?v=nsT2s40s_3k&t=414s
